cypress/e2e/billingv2.cy.ts

```diff
@@ -19,6 +23,15 @@ describe('Billing', () => {
         cy.get('[data-attr=unsubscribe-reason-survey-textarea]').type('Product analytics')
         cy.contains('.LemonModal .LemonButton', 'Unsubscribe').click()
 
+        cy.wait('@capture').then(({ request }) => {
+            const data = new Uint8Array(request.body)
+            const decoded = fflate.strFromU8(fflate.decompressSync(data))
+            const decodedJSON = JSON.parse(decoded)
+
+            // These should be a 'survey sent' event somewhere in the decodedJSON
+            const matchingEvent = decodedJSON.filter((event) => event.event === 'survey sent')
+            expect(matchingEvent).to.not.be.empty
```

Comment:
random flyby, it's a good idea to test here that the 'right' survey sent event is sent (id & response for example) - future proofs against random survey sent events & makes the test more solid.

For example, someone randomly changing the surveyID in the code should mean some tests fail.

Reply:
Thanks, Neil.